Patch to address 20638369.  This patch is Solaris specific and not
suitable for upstream contribution.

--- Django-1.8.5/django/core/management/commands/compilemessages.py.~1~	2015-10-05 02:19:35.750704280 -0800
+++ Django-1.8.5/django/core/management/commands/compilemessages.py	2015-10-05 02:20:57.589647207 -0800
@@ -34,7 +34,7 @@ class Command(BaseCommand):
     requires_system_checks = False
     leave_locale_alone = True
 
-    program = 'msgfmt'
+    program = '/usr/gnu/bin/msgfmt'
     program_options = ['--check-format']
 
     def add_arguments(self, parser):
--- Django-1.11.1/django/core/management/commands/makemessages.py	2017-07-04 12:36:33.399069813 +0000
+++ Django-1.11.1/django/core/management/commands/makemessages.py	2017-07-04 12:36:25.907294306 +0000
@@ -354,7 +354,7 @@ class Command(BaseCommand):
         if locales:
             check_programs('msguniq', 'msgmerge', 'msgattrib')
 
-        check_programs('xgettext')
+        check_programs('/usr/gnu/bin/xgettext')
 
         try:
             potfiles = self.build_potfiles()
@@ -528,7 +528,7 @@ class Command(BaseCommand):
         if self.domain == 'djangojs':
             is_templatized = build_file.is_templatized
             args = [
-                'xgettext',
+                '/usr/gnu/bin/xgettext',
                 '-d', self.domain,
                 '--language=%s' % ('C' if is_templatized else 'JavaScript',),
                 '--keyword=gettext_noop',
@@ -540,7 +540,7 @@ class Command(BaseCommand):
             ]
         elif self.domain == 'django':
             args = [
-                'xgettext',
+                '/usr/gnu/bin/xgettext',
                 '-d', self.domain,
                 '--language=Python',
                 '--keyword=gettext_noop',
